src/packages/commit-lint/core/core.js

```diff
@@ -0,0 +1,70 @@
+const lib = require('../lib');
+const { ArgumentParser } = require('@eigenspace/argument-parser');
+
+const DEFAULT_COMMIT_CONFIG_PATH = '.commit-lint.config.json';
+
+const SUCCESS_MESSAGE = '✔ Commit is valid';
+
+const ERROR_PREFIX = '✘ Commit error: \n';
+const REFERENCE_TO_DOC = 'Please, read: https://github.com/eigen-space/codestyle/tree/dev/doc/common#512-сообщение-к-изменению';
+const ISSUE_PREFIX_ERROR = `${ERROR_PREFIX} Issue prefix ":0" doesn\'t match the requirements. \n ${REFERENCE_TO_DOC}`;
+const BODIES_ERROR = `${ERROR_PREFIX} These bodies: ":0" don\'t match the requirements. \n ${REFERENCE_TO_DOC}`;
+
+function validate() {
+    const config = getConfig();
+    const commitMessage = lib.getCommitMessage();
+    const tokens = getTokensFrom(commitMessage);
+
+    checkForIgnoreMatching(config, tokens);
+    checkForIssuePrefix(config, tokens);
+    checkForBodies(config, tokens);
+
+    lib.exitWithSuccess(SUCCESS_MESSAGE);
+}
+
+function getConfig() {
+    const parser = new ArgumentParser();
+    const args = parser.get(process.argv.slice(2));
+
+    const configPath = args.get('config') || DEFAULT_COMMIT_CONFIG_PATH;
+
+    const sourceConfig = lib.getConfigContentFrom(configPath);
```

Comment:
It seems to be a part of `lib.getConfigContentFrom`, i.e. it is better if function returns requested config file resolving its configuration like `extends`.

Reply:
Planned to the next release